### ISSUE TITLE
Docker CLI

### ISSUE THREAD

**Issue by Clockwork-Muse** [microsoft/winget-pkgs#27183]
# Package Requested

With the changes to [Docker's licensing terms](https://www.docker.com/blog/updating-product-subscriptions/), many teams may be moving away from Docker Desktop itself, and towards downloading only the CLI.  Too, this would be a far smaller install package if the local installation was only ever used to manage remote docker hosts.

- [x] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [x] I have performed a search and couldn't find this package.
- [ ] I think there is a new version available and I have provided the URL.

---

## Please provide the following information if you can:

**Publisher**: Docker Inc.

**Publisher Url**: https://www.docker.com

**Package Name**: Docker CLI

**Package Url**: https://www.docker.com

**Description**: The Docker CLI is an application for MacOS and Windows machines to access remote Docker hosts for the building and sharing of containerized applications. Access a remote host via the Docker CLI and follow the guided onboarding to build your first containerized application in minutes.

**Package Version**: 20.10.8 (as of this writing)

**Installer URL**:

There's a similar (although not completely up-to-date) [package in the chocolatey repository](https://community.chocolatey.org/packages/docker-cli).

Recent docker builds, including the CLI, [are available](https://download.docker.com/win/static/stable/x86_64), however this isn't all versions, and it's not completely clear whether the use of these builds would be exempt from the licensing restrictions.  Otherwise, a custom build of the CLI itself [from the main repository](https://github.com/docker/cli) will be required.

**Comment by Clockwork-Muse** [comment]
This is distinct from the main `DockerDesktop` package, in that it would contain only the CLI.

**Comment by jedieaston** [comment]
It's going to require portable application support, unfortunately. This is blocked by https://github.com/microsoft/winget-cli/issues/140.

**Comment by jedieaston** [comment]
(Also, for anyone looking at this issue in the future, Podman is blocked by .zip installation too and would be nice to have: https://podman.io/getting-started/installation).

**Comment by Clockwork-Muse** [comment]
@jedieaston - only if the packages from docker.com itself are used.  The actual build scripts likely don't output archives (although separate infrastructure becomes its own issue).

**Comment by jedieaston** [comment]
The build scripts don't build installers, do they?

**Comment by Clockwork-Muse** [comment]
I honestly don't know, and haven't looked into it.  The downloads I listed earlier appear to only be binaries as well, without an install script.

**Comment by Trenly** [comment]
@msftbot .zip  - This is mislabeled as .exe

**Comment by mdanish-kh** [comment]
Close with reason: Package is now available under PackageId Docker.DockerCLI;

https://github.com/microsoft/winget-pkgs/tree/master/manifests/d/Docker/DockerCLI

**Comment by jankap** [comment]
Any chance to get buildkit integrated?

```
docker buildx
docker: unknown command: docker buildx

Run 'docker --help' for more information
```

Installing it is quite simple, see https://github.com/docker/buildx. Just download, rename and put to `%userprofile%/.docker/cli-plugins`.
